index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+    <!--google fonts-->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+    <!--stylesheet-->
+    <link rel="stylesheet" href="style.css">
+
+    <title>Portfolio</title>
+</head>
+<body>
+    <!--nav-->
+    <nav>
+        <a href="#">Akshitha Reddy</a>
+        <button class="menu-container"><img src="./assest/Enabled.png" alt="menu"></button>
+    </nav>
```

Comment:
- [ ] Please put the `nav` tag with its child elements into a `<header>` tag. This way the HTML will be more semantic 🤗